src/core/event/index.js

```diff
@@ -296,13 +296,20 @@ export function Events(Base) {
     // =========================================================================
     /**
      * Handle rendering UI element updates and new content
+     * @param {string} name the site name
      * @void
      */
-    onRender() {
+    onRender(name) {
       const currentPath = this.router.toURL(this.router.getCurrentPath());
```

Comment:
Why not use it like this?

```suggestion
    onRender() {
      const { name } = this.config;
      const currentPath = this.router.toURL(this.router.getCurrentPath());
```

Reply:
update.